rslib/src/deckconfig/update.rs

```diff
@@ -171,20 +173,74 @@ impl Collection {
                 };
 
                 // if new order differs, deck needs re-sorting
-                let current_order = configs_after_update
-                    .get(&current_config_id)
+                let current_config = configs_after_update.get(&current_config_id);
+                let current_order = current_config
                     .map(|c| c.inner.new_card_insert_order())
                     .unwrap_or_default();
                 if previous_order != current_order {
                     self.sort_deck(deck_id, current_order, usn)?;
                 }
+
+                record_steps_len_delta(
+                    previous_config,
+                    current_config,
+                    &mut decks_by_steps_len_delta,
+                    deck_id,
+                );
             }
         }
 
+        self.adjust_remaining_learning_steps(&decks_by_steps_len_delta)?;
         self.set_config_string_inner(StringKey::CardStateCustomizer, &input.card_state_customizer)?;
 
         Ok(())
     }
+
+    fn adjust_remaining_learning_steps(
+        &mut self,
+        decks_by_steps_len_delta: &HashMap<i32, Vec<DeckId>>,
+    ) -> Result<()> {
+        let usn = self.usn()?;
+        for (&delta, decks) in decks_by_steps_len_delta {
+            for mut card in self.all_learning_cards_in_decks(decks)? {
+                let original = card.clone();
+                // strip "remaining today"
+                let remaining = (card.remaining_steps % 1000) as i32;
```

Comment:
I understand `remaining_steps / 1000` is the number of learning steps that can be completed before the day rolls over in v2. Is this still used? I couldn't find this value being updated after a config change, so I reasoned it wouldn't aggrevate things to simply strip it.